src/Spatial/Units/Angle.cs

```diff
@@ -311,6 +335,44 @@ public static Angle ReadFrom(XmlReader reader)
             return reader.ReadElementAs<Angle>();
         }
 
+        /// <summary>
+        /// Gets the sine of the angle
+        /// </summary>
+        /// <returns>The sine of the angle</returns>
+        public double Sin()
```

Comment:
Not sure about this `a.Cos()` reads a bit weird to me.

Reply:
if I have created an angle, the idea that I have to turn it back into a double with myangle.Radians in order to get the sin or cos of the angle really doesn't encourage me to use Angle at all.  I start asking why does it exist?  Also from a simplicity point of view I find myangle.Sin() much more streamlined than Math.Sin(myangle.Radians);

Reply:
Angle is a very nice abstraction imo, I like types. When used as a parameter we make it explicit that it is an angle avoiding one class of bugs, passing the wrong thing. Also if we used doubles our parameter names would be `angleInRadians` and it would still be easy to make a mistake an pass in degrees.

`Math.Sin(myangle.Radians)` is nice and explicit imo. I don't know if extension everything in C#8 will allow us to add extension methods to static classes, if so adding overloads to `Math` taking `Angle`could be nice.

Reply:
I must admit I am confused that just above you agree that because there is an ambiguity between using degrees and using radians you prefer not to have explicit conversion but here you are comfortable with the fact that Math.Sin requires you to be aware that you need to pass in radians and not degrees and find this sufficient.

Reply:
`Sin` as a member on angle has weird semantics to me. We can wait and see if other people have different opinion. Also I checked with the roslyn guys, looks like we will be able to add static overloads to `Math` that accepts our `Angle`